src/ImageSharp.Web/Helpers/FormatHelpers.cs

```diff
@@ -34,14 +36,25 @@ public static string GetContentType(Configuration configuration, string key)
         /// <returns>The <see cref="string"/></returns>
         public static string GetExtension(Configuration configuration, string uri)
         {
+            string[] parts = uri.Split('?');
+            if (parts.Length > 1)
+            {
+                StringValues ext;
+                if (QueryHelpers.ParseQuery(parts[1]).TryGetValue("format", out ext))
```

Comment:
Should use `FormatWebProcessor.Format` rather than the inline string. We can inline that out param also to keep it pretty.

Reply:
Done! :)